src/CMakeLists.txt

```diff
@@ -196,12 +196,7 @@ find_package(CapnProto REQUIRED)
 include_directories(${CAPNP_INCLUDE_DIRS})
 # List all .capnp files here. The C++ files will be generated and included
 # when compiling later on.
-set(CAPNP_SPECS
-    nupic/algorithms/SpatialPoolerProto.capnp
-    nupic/math/SparseBinaryMatrixProto.capnp
-    nupic/math/SparseMatrixProto.capnp
-    nupic/utils/RandomProto.capnp
-)
+file(GLOB CAPNP_SPECS nupic/proto/*.capnp)
```

Comment:
@oxtopus - so it recommends listing out individual files or is there a way to specify all files in a given directory that doesn't suffer the same issue?